pykubectl/kubectl.py

```diff
@@ -20,6 +20,9 @@ def execute(self, command, definition=None, safe=False):
 
             logging.debug(f'executing {cmd}')
 
+            if validate:
+                cmd = f"{cmd} --dry-run --validate"
```

Comment:
the semantic of using both dry-run and validate at the same time are not clear to me. do you know?

Reply:
Looks like validate will still affect the change but just validates it first, while dry-run won't:
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply

Reply:
Per https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply

```suggestion
                cmd = f"{cmd} --dry-run=client --validate"
```

Reply:
so validate is probably on by default. I don't think needs to be specified